server/game/cards/06-CotE/SageOfGiseiToshi.js

```diff
@@ -0,0 +1,24 @@
+const DrawCard = require('../../drawcard.js');
+const { Players, CardTypes } = require('../../Constants');
+
+class SageOfGiseiToshi extends DrawCard {
+    setupCardAbilities(ability) {
+        this.action({
+            title: 'Move home, then move character home',
+            condition: context => this.game.isDuringConflict() && context.player.opponent && context.player.opponent.honor < context.player.honor,
+            gameAction: ability.actions.sendHome(),
+            target: {
+                cardType: CardTypes.Character,
+                controller: Players.Opponent,
+                cardCondition: (card) => card.isParticipating()
```

Comment:
I think we need to check `card.allowGameAction('moveHome', context)` here as well. Normally the game will do that automatically, but I don't think it will where the target is only affected by the post-then part of the effect

Reply:
Done.